TODO.md

```diff
@@ -28,6 +28,11 @@ see CHANGELOG.md for a more formal list of changes by release
 * gui, add confirmation before deleting addon directory
     - language should be 'remove' rather than 'delete'
 
+* reconciliation, revisit aliases
+    - use source and source-id now
+        - done
+    - maybe externalise the list
```

Comment:
nah, but it could do with an update. I think the last time I downloaded the 50 most popular addons, deleted the nfo file and aliased the ones that didn't reconcile.

Reply:
updated after writing a script to find the *recently* popular vs the *all time* popular (see ogri-la/strongbox-catalogue).

Surprisingly few and they fall into two camps: the .toc `Title` is a shorted version of the catalogue title and marked up text.